Source/SourceKittenFramework/Request.swift

```diff
@@ -57,31 +57,31 @@ private func fromSourceKit(_ sourcekitObject: sourcekitd_variant_t) -> SourceKit
     switch sourcekitd_variant_get_type(sourcekitObject) {
     case SOURCEKITD_VARIANT_TYPE_ARRAY:
         var array = [SourceKitRepresentable]()
-        _ = sourcekitd_variant_array_apply(sourcekitObject) { index, value in
-            if let value = fromSourceKit(value) {
-                array.insert(value, at: Int(index))
-            }
-            return true
+        _ = withUnsafeMutablePointer(to: &array) { arrayPtr in
+            sourcekitd_variant_array_apply_f(sourcekitObject, { index, value, context in
+                if let value = fromSourceKit(value), let context = context {
+                    let localArray = context.bindMemory(to: [SourceKitRepresentable].self, capacity: 1)
```

Comment:
Sorry for late notice.
I guess that this can be `assumingMemoryBound(to:)` instead of `bindMemory(to:capacity:)`, because `context` is already initialized before calling `withUnsafeMutablePointer(to:)`.